src/main/scala/org/deepdive/ddlog/DeepDiveLogPrettyPrinter.scala

```diff
@@ -152,8 +162,7 @@ object DeepDiveLogPrettyPrinter extends DeepDiveLogHandler {
 
   def print(stmt: InferenceRule): String = {
     print(stmt.q) +
-    ( if (stmt.weights == null) ""
-      else "\n  weight = " + (stmt.weights.variables.map(print).mkString(", "))
+    ( "\n  weight = " + (stmt.weights.variables.map(print).mkString(", "))
```

Comment:
Nice.  Seems like the null check was meaningless here.  Can we use the style without dots in the chained method calls for consistency?
```scala
stmt.weights.variables map print mkString(", ")
```